pandas/tests/io/json/test_pandas.py

```diff
@@ -642,6 +642,13 @@ def test_series_from_json_precise_float(self):
         result = read_json(s.to_json(), typ='series', precise_float=True)
         assert_series_equal(result, s, check_index_type=False)
 
+    def test_series_with_dtype(self):
```

Comment:
This test should hit the modification made here: https://github.com/mroeschke/pandas/blob/0d7f07783ad9a42bb3fc7f0e3dda0c01b877fe57/pandas/io/json/json.py#L550